README.md

```diff
@@ -52,4 +52,5 @@ If you want to start building an Operator, you should definitely look into the [
 | Prometheus Jmx Exporter | [banzaicloud/prometheus-jmx-exporter-operator](https://github.com/banzaicloud/prometheus-jmx-exporter-operator) | This operator using Jmx Exporter enables Java processes running ok Kubernetes Pods to expose metrics collected form mBeans via JMX to Prometheus. |
 | Envoy | [solo-io/envoy-operator](https://github.com/solo-io/envoy-operator)| Run and manage Envoy on Kubernetes simply and securely. |
 | CloudFormation | [linki/cloudformation-operator](https://github.com/linki/cloudformation-operator)| Run and manage CloudFormation stacks. Manage AWS resources from kubernetes. |
+| Vault Operator | [banzaicloud/bank-vaults](https://github.com/banzaicloud/bank-vaults)| A feature rich HA Vault operator with TLS, external API based re/configuration, several/automatic unseal options and more.|
```

Comment:
Would you move this up right below the other Vault Operator from CoreOS? It would be good to follow other patterns and change both to prefix with a number such as `Vault #1` and `Vault #1`.